Create SJET-AOCODAF405V3.config

634296207 opened the pull request:
Create SJET-AOCODAF405V3.config

634296207 committed:
Create SJET-AOCODAF405V3.config

Create SJET-AOCODAF405V3.config

lida2003 commented:
LGTM, please do self test. Make sure self-test is passed, and add comment below

```
F405V3 Flash V2/V3 self test passed.
```

634296207 committed:
Update SJET-AOCODAF405V3.config

1. Select GPS for serial port 1 and vtx for serial port 2

2.LED lights are turned on by default

3. The default gps protocol is ublox

4. The receiver selects crsf protocol by default

5. The default motor protocol is Dshot600

6. Add user2 mode,

7. Bluetooth and vx (9v) are enabled by default

lida2003 commented on configs/default/SJET-AOCODAF405V3.config:
```diff
@@ -0,0 +1,147 @@
+
+# serial
+serial 1 64 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
```
Why there are two definitions for serial 1 "serial 1 64 115200 57600 0 115200"?

634296207 committed:
Update SJET-AOCODAF405V3.config

1. Delete duplicate serial ports

2. Serial port 0 Select GPS,

634296207 commented:
F405V3 Flash V2/V3 self test passed.

634296207 commented:
F405V3 Flash V2/V2 self test passed.

lida2003 commented:
Add default features, which should be added when PR is created.

Just copied from https://github.com/Aocoda-RC/unified-targets/pull/2/commits/f2a0bfab73daed7f8c12c9b5f2a9c24e2ff34ffc
---

1. Select GPS for serial port 1 and vtx for serial port 2
2.LED lights are turned on by default
3. The default gps protocol is ublox
4. The receiver selects crsf protocol by default
5. The default motor protocol is Dshot600
6. Add user2 mode,
7. Bluetooth and vx (9v) are enabled by default
